doc/rst/tools/mason/mason.rst

```diff
@@ -72,28 +60,26 @@ following hierarchy::
 
 Mason will enforce that the main file be named after the package to enforce namespacing.
 While it is common practice for package names to be PascalCase and chpl files to be lowercase,
-it is an acceptable tradeoff for reliability, and MyPackage will be the first file listed in ``src/``.
-
-
+it is an acceptable tradeoff for reliability. MyPackage will be the first file listed in ``src/``.
```

Comment:
@ben-albrecht we dont do that anywhere else we say MyPackage so should that be a global change?